src/js/components/scene-selector.jsx

```diff
@@ -99,6 +104,35 @@ var SceneSelector = React.createClass({
     this.setState({showModal: false})
   },
 
+  //details modal
+
+  _showSceneDetailsModal: function() {
+    if (this.state.sceneListFilter.value != "none") {
+      this.setState({showDetailsModal: true})
+    }
+  },
+
+  _closeDetailsModal: function () {
+    this.setState({showDetailsModal: false})
+  },
+
+  _completeDetailsModal: function() {
+    var newName = this.sceneNameEditInput.value;
+    var self = this;
+    //potentialy risky (saving over scene)
+    SceneActions.getFullScene(this.state.sceneListFilter.value._id, function(scene) {
```

Comment:
I think this is the wrong call to make, the Full Scene call goes to the asset store to get the database mos appended to it.  

I think using the HubClient - loadSceneWithCb method is a better choice, as it requests the Scene JSON that we store in the scene collection.  The asset store call pulls this scene then adds properties to each mo individually if a record is in either the video, audio or image collections.  We don't actually store this full scene representation for now.

Reply:
Make's sense, changed to use the HubClient for load and save during name change.